test/python/test_composite.py

```diff
@@ -38,26 +39,26 @@ def test_load_thermo_models(self):
                 z = sol.state # calls Phase::saveState
                 sol.TP = 300, 2*ct.one_atm
                 sol.state = z # calls Phase::restoreState
-                self.assertEqual(sol.T, T0)
-                self.assertEqual(sol.P, p0)
+                assert sol.T == T0
+                assert sol.P == p0
 
                 if sol.thermo_model in ('pure-fluid',):
-                    self.assertTrue(sol.has_phase_transition)
+                    assert sol.has_phase_transition
                 else:
-                    self.assertFalse(sol.has_phase_transition)
+                    assert not sol.has_phase_transition
 
                 if not sol.is_compressible:
-                    with self.assertRaisesRegex(ct.CanteraError,
-                                                'Density is not an independent'):
+                    with pytest.raises(ct.CanteraError,
+                                                match='Density is not an independent'):
```

Comment:
Please re-indent in cases like this:
```suggestion
                    with pytest.raises(ct.CanteraError,
                                       match='Density is not an independent'):
```